src/DynamoCoreWpf/Views/PackageManager/Controls/PackageManagerPackagesControl.xaml.cs

```diff
@@ -89,6 +90,34 @@ private void ViewDetailsButton_OnClick(object sender, RoutedEventArgs e)
             if (!(button.DataContext is PackageManagerSearchElementViewModel packageManagerSearchElementViewModel)) return;
 
             var PkgSearchVM = this.DataContext as PackageManagerSearchViewModel;
+
+            var name = this.Name;
+            if (name.Equals(Dynamo.Wpf.Properties.Resources.PackageManagerSearchPackagesControlName))
```

Comment:
It's not a common practice to store the name of the control as resources because they will get localized, unless that was your intention? Maybe we can use static readonly strings

Reply:
If that's ok, I will remove the redundant localized strings in the next PR, as there are quite a few changes in the resources files lined up.

Reply:
sure

Reply:
Removed the resource with this commit https://github.com/DynamoDS/Dynamo/pull/14299/commits/cc58820dadaddfbb90b602f69fc009c55361be72